core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/DataSource.scala

```diff
@@ -30,6 +30,34 @@ class SlickDataSource(val databaseConfig: DatabaseConfig[JdbcProfile])(implicit
     database.run(f(dataAccess).transactionally.withTransactionIsolation(isolationLevel))
   }
 
+  def createTempTable = {
+    sql"""call createEntityAttributeTempTable()""".as[Boolean]
+  }
+
+  def dropTempTable = {
+    sql"""call dropEntityAttributeTempTable""".as[Boolean]
```

Comment:
these queries seem to return the same value, regardless of whether or not the table exists. i think the only way to check for the existence of a temp table would be to write a stored procedure to do so